e2e-tests/quote/cypress/support/commands.js

```diff
@@ -0,0 +1,145 @@
+Cypress.Commands.add('completeAndSubmitDeclarationHowYourDataWillBeUsed', require('../../../commands/insurance/declarations/complete-and-submit-how-your-data-will-be-used-form'));
+
+Cypress.Commands.add('completeAndSubmitDeclarations', require('../../../commands/insurance/complete-declarations'));
+Cypress.Commands.add('completeSignInAndSubmitAnApplication', require('../../../commands/insurance/complete-sign-in-and-submit-an-application'));
+
+Cypress.Commands.add('assertAllSectionsUrl', require('../../../commands/insurance/assert-all-sections-url'));
+
+Cypress.Commands.add('assertChangeAnswersPageUrl', require('../../../commands/insurance/assert-change-answers-page-url'));
+Cypress.Commands.add('assertSummaryListRowValue', require('../../../commands/assert-summary-list-row-value'));
+Cypress.Commands.add('assertSummaryListRowValueNew', require('../../../commands/assert-summary-list-row-value-new'));
+Cypress.Commands.add('submitAndAssertRadioErrors', require('../../../commands/submit-and-assert-radio-errors'));
+Cypress.Commands.add('submitAndAssertFieldErrors', require('../../../commands/submit-and-assert-field-errors'));
+
+Cypress.Commands.add('assertPasswordLabelHintAndInput', require('../../../commands/insurance/account/assert-password-label-hint-and-input'));
+Cypress.Commands.add('assertPasswordRevealButton', require('../../../commands/insurance/account/assert-password-reveal-button'));
+Cypress.Commands.add('assertConfirmEmailPageContent', require('../../../commands/insurance/account/assert-confirm-email-page-content'));
+
+Cypress.Commands.add('assertSubmitAndSaveButtons', require('../../../commands/insurance/assert-submit-and-save-buttons'));
+
+Cypress.Commands.add('assertUncheckedYesNoRadios', require('../../../commands/assert-unchecked-yes-no-radios'));
+
+Cypress.Commands.add('assertCustomerServiceContactDetailsContent', require('../../../commands/assert-customer-service-contact-details-content'));
+
+Cypress.Commands.add('checkText', require('../../../commands/check-text'));
+Cypress.Commands.add('checkValue', require('../../../commands/check-value'));
+Cypress.Commands.add('checkAriaLabel', require('../../../commands/check-aria-label'));
+Cypress.Commands.add('checkRadioInputYesAriaLabel', require('../../../commands/check-radio-input-yes-aria-label'));
+Cypress.Commands.add('checkRadioInputNoAriaLabel', require('../../../commands/check-radio-input-no-aria-label'));
+
+Cypress.Commands.add('checkTaskStatus', require('../../../commands/check-task-status'));
+Cypress.Commands.add('checkTaskStatusCompleted', require('../../../commands/check-completed-task-status'));
+Cypress.Commands.add('checkLink', require('../../../commands/check-link'));
+Cypress.Commands.add('getReferenceNumber', require('../../../commands/get-reference-number'));
+Cypress.Commands.add('keyboardInput', require('../../../commands/keyboard-input'));
```

Comment:
Commands seem to be duplicated in the insurance and quote directories. Are we able to be DRY?

Reply:
done